docs/user/reference/metrics/index.md

```diff
@@ -66,17 +66,7 @@ Labeled metrics come in two forms:
 
 ### Label format
 
-To ensure maximum support in database columns, labels must be made up of dot-separated identifiers with lowercase ASCII alphanumerics, containing underscores and dashes.
-
-Specifically, they must conform to this regular expression:
-
-```
-^[a-z_][a-z0-9_-]{0,29}(\\.[a-z_][a-z0-9_-]{0,29})*$
-```
-
-Check your label:
-<input type="text" label="Label" id="label">
-<span id="result">unchecked</span>
+Labels must not exceed 60 characters in length, and may comprise any ASCII characters.
```

Comment:
*me starts putting null bytes in there*

Reply:
Your proposal calls out "Or at least everything printable." Maybe we should go for that.